opennms-webapp/src/main/java/org/opennms/web/controller/event/EventController.java

```diff
@@ -96,6 +104,9 @@
 
     private boolean m_showEventCount = false;
 
+    private RedirectRestricter redirectRestricter = RedirectRestricter.builder()
+            .build();
```

Comment:
Does this actually work?  As far as I can tell you didn't pass a set of allowed redirects into the builder, so it would just reject everything.